modules/bridge/src/main/ts/ephox/bridge/components/dialog/DialogFooterButton.ts

```diff
@@ -77,10 +73,7 @@ const baseFooterButtonFields = [
   ComponentSchema.generatedName('button'),
   ComponentSchema.optionalIcon,
   FieldSchema.defaultedStringEnum('align', 'end', [ 'start', 'end' ]),
-  // this should be removed, but must live here because FieldSchema doesn't have a way to manage deprecated fields
-  ComponentSchema.primary,
   ComponentSchema.enabled,
-  // this should be defaulted to `secondary` but the implementation needs to manage the deprecation
   FieldSchema.optionStringEnum('buttonType', [ 'primary', 'secondary' ])
```

Comment:
Is the comment above saying this should be changed to the following?
```suggestion
  FieldSchema.defaultedStringEnum('buttonType', 'secondary', [ 'primary', 'secondary' ])
```

Reply:
That is very possible. I'll change it.